plugins/rbac/src/components/EditRole.tsx

```diff
@@ -29,7 +29,7 @@ const EditRole = ({
         <IconButton
           color="inherit"
           component={Link}
-          aria-label="Update"
+          aria-label={dataTestId}
```

Comment:
we shouldn't update aria-label like this, it should be alternative text used for accessibility